lib/posix/pthread.c

```diff
@@ -215,39 +215,40 @@ void __z_pthread_cleanup_pop(int execute)
 	}
 }
 
-static bool is_posix_policy_prio_valid(uint32_t priority, int policy)
+static bool is_posix_policy_prio_valid(int priority, int policy)
```

Comment:
The `int sched_get_priority_min()` function always returns 0, if `policy` is valid.